src/style.css

```diff
@@ -0,0 +1,97 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+.block_container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-basis: 300px;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  width: 100%;
```

Comment:
Hi. If  I delete the value `width: 100%`, I can not pass the 450 screen expansion tests. Because the blocks are stretched only to the minimum width specified in min-width: 300px; and are shaped into squares, not rectangles, and do not occupy the entire width of the screen, but only the specified minimum width. 
Therefore, I left this value, as well as the value of the minimum width of the block, because it is a condition of the task.All other errors have been fixed http://joxi.ru/brR3gX5FBEDv0m

Reply:
Try to fix it by using `flex-grow` and `flex-basis`